src/utils/types.ts

```diff
@@ -1 +1,10 @@
 export type Nullable<T> = T | null;
+
+export interface SignUpValues {
```

Comment:
this is a form values interface only used on signup, keep more general interfaces here.